push/forms.py

```diff
@@ -7,6 +7,10 @@ class PushAppForm(ModelForm):
     class Meta:
         model = PushApplication
         fields = ['name', 'vapid_key']
+        widgets = {
+            'name': TextInput(attrs={'required': True}),
+            'vapid_key': TextInput(attrs={'required': True})
+        }
```

Comment:
At one point, I cautioned @openjck against adding `widgets` to our forms, because it seemed like HTML-specific code should live in the HTML - not in the python code.

So, I'd be happy to change to that design - i.e., change the `applications.html` (soon `list.html`) template to manually create the `input` elements rather than using Django's form widgets.